llvm/lib/TargetParser/SubtargetFeature.cpp

```diff
@@ -81,5 +82,7 @@ void SubtargetFeatures::getDefaultSubtargetFeatures(const Triple& Triple) {
       AddFeature("64bit");
       AddFeature("altivec");
     }
+  } else if (Triple.isAndroid() && TargetABI.contains("lp64d")) {
```

Comment:
Do we have stability guarantees about the `target-abi` metadata? I don't think we do, but I also don't recall any changes around it... Is relying on parsing the string stable?

Reply:
Should we check that the triple is RISCV?

Reply:
> Do we have stability guarantees about the target-abi metadata? I don't think we do, but I also don't recall any changes around it... Is relying on parsing the string stable?

I think there was a patch that added lp* information as metadata in lto/thinlto. So far all the examples I saw also had this information but I can look up the patch if that is needed. At least for the use case I'm looking at this change should be sufficient.

Reply:
> Should we check that the triple is RISCV?

I'm okay with it. Preparing the patch now.